packages/api/src/sequelize/migrations/2024-11-19_00_update-patient-add-opting-out.ts

```diff
@@ -0,0 +1,24 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+
+const patientTableName = "patient";
+const optingOutColumn = "opting_out";
+
+export const up: Migration = async ({ context: queryInterface }) => {
+  return queryInterface.sequelize.transaction(async transaction => {
+    await queryInterface.addColumn(
+      patientTableName,
+      optingOutColumn,
+      { type: DataTypes.BOOLEAN, defaultValue: false },
```

Comment:
Always indicate whether the column is required or not - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#86b6af7b42214698877e55e0733cf9c2)